tests/spec/core/biblio-spec.js

```diff
@@ -209,3 +209,37 @@ describe("W3C — Bibliographic References", () => {
     expect(refs[0].textContent).toEqual("[dom]");
   });
 });
+
+it("makes sure the Informative references section has expected localization text", async () => {
+  const ops = {
+    config: makeBasicConfig(),
+    htmlAttrs: {
+      lang: "nl",
+    },
+    body: `<section class="informative" id="intro">[[DOM]]</section>`,
+  };
+  const doc = await makeRSDoc(ops);
+  const ref = doc.getElementById("references");
```

Comment:
Here, you don't need ref, just grab the h2's textContent directly. 

```suggestion
  const { textContent } = doc.getElementById("#references h2");
```

Reply:
I tried this too...seems not to work with the test spec I applied to `body`. But I will try to figure out why and apply your suggestions now. Thanks.

Reply:
Sorry, should be: 
const { textContent } = doc.querySelector("#references h2");